ngraph/core/reference/include/ngraph/runtime/reference/convolution.hpp

```diff
@@ -42,15 +40,18 @@ namespace ngraph
                     std::vector<int> dilation;
                     std::vector<int> pads_begin;
                     std::vector<int> pads_end;
+                    std::vector<int> output_padding;
```

Comment:
Please correct me, but if I remember right convolution doesn't have output_padding.

Reply:
Yes, it is correct, convolution does not have `output_padding` attribute. I introduced it into convolution in order to reuse convolution implementation for convolution backprop (both convolutions share the same attributes, but backprop additionally uses output_padding). `output_padding` is set to 0's as a default value so it won't impact on convolution implementation.